contracts/test/Rollup.t.sol

```diff
@@ -0,0 +1,426 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * Modifications Copyright 2022, Specular contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../src/ISequencerInbox.sol";
+import "../src/libraries/Errors.sol";
+import {SpecularProxy} from "test/SpecularProxy.sol";
+import {MockToken} from "./utils/MockToken.sol";
+import {Utils} from "./utils/Utils.sol";
+import {IRollup} from "../src/IRollup.sol";
+import {Verifier} from "../src/challenge/verifier/Verifier.sol";
+import {Rollup} from "../src/Rollup.sol";
+import {SequencerInbox} from "../src/SequencerInbox.sol";
+import {RLPEncodedTransactionsUtil} from "./utils/RLPEncodedTransactions.sol";
+
+contract RollupBaseSetup is Test, RLPEncodedTransactionsUtil {
+    Utils internal utils;
+    address payable[] internal users;
+
+    address internal sequencer;
+    address internal alice;
+    address internal bob;
+    address internal vault;
+    address internal sequencerOwner;
+    address internal sequencerAddress;
+    address internal rollupOwner;
+
+    Verifier verifier = new Verifier();
+
+    function setUp() public virtual {
+        utils = new Utils();
+        users = utils.createUsers(3);
+
+        sequencer = users[0];
+        vm.label(sequencer, "Sequencer");
+
+        alice = users[1];
+        vm.label(alice, "Alice");
+
+        bob = users[2];
+        vm.label(bob, "Bob");
+
+        vault = makeAddr("vault");
+        sequencerOwner = makeAddr("sequencerOwner");
+        sequencerAddress = makeAddr("sequencerAddress");
+        rollupOwner = makeAddr("rollupOwner");
```

Comment:
why two ways of creating users? please clean up

Reply:
The reason was it was easier for me to move forward by just using the cheat `makeAddr`,  since that would mean no extra *utils* file required.
But for some reason, it was not possible for me to change their Ether balance. Therefore, I have reverted back to the old method of creating new users.

Also removed the SpecularProxy.

Please review again. Thank you.